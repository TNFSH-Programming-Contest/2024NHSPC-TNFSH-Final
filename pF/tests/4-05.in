977301286378 976367
