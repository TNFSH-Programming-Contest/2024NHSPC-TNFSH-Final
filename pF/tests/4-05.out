488183331383914969